share/completions/iwctl.fish

```diff
@@ -45,7 +59,23 @@ function __iwctl_connect
     # remove all options
     argparse -i 'u/username=' 'p/password=' 'P/passphrase=' 'v/dont-ask' -- $argv
     # station name should now be the third argument (`iwctl station <wlan>`)
-    __iwctl_filter station $argv[3] get-networks
+    for network in (__iwctl_filter station $argv[3] get-networks rssi-dbms --all-columns)
+        set network (string split \t $network)
+        set -l strength $network[3]
+        # This follows iwctls display of * to ****
+        # https://git.kernel.org/pub/scm/network/wireless/iwd.git/tree/client/station.c?id=4a0a97379008489daa108c9bc0a4204c1ae9c6a8#n380
+        if test $strength -ge -6000
```

Comment:
Oh, sure, but it's a good habit to get into.

In this case you could use

```fish
if test "0$strength" -ge -6000
```

and it would just give you a strength of 4 if network is the wrong format - because with an empty $strength that would turn into `test 0 -ge -6000`

Reply:
> In this case you could use
> 
> ```fish
> if test "0$strength" -ge -6000
> ```

Wouldn't work, because `strength` is negative.

Reply:
That's why it would give a strength of *4* (i.e. `****`) - the first test would be *true*.

My thinking was that at that point things are already wrong, so in completions (where it's all trying to get something anyway) that would be one way to handle it.

Reply:
but 0-6500 would not be valid

Reply:
Aaaah, that's of course correct.